client/src/app/shared/shared-instance/instance-features-table.component.scss

```diff
@@ -7,7 +7,7 @@ table {
 
   .label,
   .sub-label {
-    min-width: 330px;
+  
```

Comment:
The lines 10 and 11 should be deleted.

Reply:
@kimsible hey. do i create a new pull request  and close this one? i removed the spaces in 10 and 11

Reply:
@kimsible yeah. so the second change shows up here so i think i dont have to create a new pull request.